uber/templates/emails/shifts/schedule.html

```diff
@@ -5,23 +5,28 @@
 {{ attendee.first_name }},
 <br/> <br/>
 
-{% if attendee.badge_type == c.STAFF_BADGE %}    
+{% if attendee.badge_type == c.STAFF_BADGE %}
 Thanks again for Staffing at {{ c.EVENT_NAME }}!
 {% endif %}
-    
+
 {% if c.VOLUNTEER_RIBBON in attendee.ribbon_ints %}
 Thanks again for Volunteering at {{ c.EVENT_NAME }}!
 {% endif %}
 <br/> <br/>
 
-{% if attendee.badge_type == c.STAFF_BADGE %}   
-- Please remember to also pick up your TWO {{ c.EVENT_NAME }} Staff Shirts from the MAGFest MERCH Booth when you get to {{ c.EVENT_NAME }}.  As a reminder, there is an expectation that you will wear one of these two shirts while working your scheduled shifts during {{ c.EVENT_NAME }}, (Thurs-Sunday, not during load-in or load-out).  When not on shift, you should wear a different shirt, so you can enjoy MAGFest 2017 with everyone else.    
+{% if attendee.badge_type == c.STAFF_BADGE and (c.SHIRTS_PER_STAFFER > 0 or c.STAFF_ELIGIBLE_FOR_SWAG_SHIRT) %}
+- Please remember to also pick up your {% if c.SHIRTS_PER_STAFFER > 1 %}{{ c.SHIRTS_PER_STAFFER }}{% endif %}
+{{ c.EVENT_NAME }} {% if c.SHIRTS_PER_STAFFER > 0 %}Staff Shirt{{ c.SHIRTS_PER_STAFFER|pluralize }}{% else %}Shirt{% endif %}
+from the {{ c.EVENT_NAME }} MERCH Booth when you get to {{ c.EVENT_NAME }}.  As a reminder, there is an expectation that you will wear
+{% if c.SHIRTS_PER_STAFFER > 1 %}one of these shirts{% else %}this shirt{% endif %} while working your scheduled shifts
+during {{ c.EVENT_NAME }}, (Thurs-Sunday, not during load-in or load-out).  When not on shift, you should wear a different
+shirt, so you can enjoy {{ c.EVENT_NAME }} with everyone else.
 {% endif %}
```

Comment:
We may also want to consider adding text here to accommodate a situation where staff get both a swag shirt AND staff shirts. This isn't true for any of our events right now, but it covers all our bases for the future. I think the if-check would look something like `if c.SHIRTS_PER_STAFFER > 0 and c.STAFF_ELIGIBLE_FOR_SWAG_SHIRT` and the text could say something like "You also qualify for (an) {{ c.EVENT_NAME }} swag shirt(s). This is the same shirt that attendees can buy and is provided as a perk for staffing."